src/app/window-handler.ts

```diff
@@ -233,6 +232,13 @@ export class WindowHandler {
     this.isCustomTitleBar =
       isWindowsOS &&
       this.config.isCustomTitleBar === CloudConfigDataTypes.ENABLED;
+    // Get url to load from cmd line or from global config file
+    const urlFromCmd = getCommandLineArgs(process.argv, '--url=', false);
+    this.isPodConfigured = !config.isFirstTimeLaunch();
+    this.didShowWelcomeScreen = false;
+    this.shouldShowWelcomeScreen =
```

Comment:
I gave it a try on st3 (sso not being configured) and I believe we should also check if the pod has sso. If SSO is not configured, we should proceed with normal flow and add log something about it.

About SSO:
- we can get SSO status by relying on `/login/checkauth?type=user` endpoint
- this endpoint response contains a field `authenticationType`. If `authenticationType` === `password` then we can skip opening the browser